### PR TITLE
chore: add `Button as div` example

### PR DESCRIPTION
**Pull request by saideepesh000**
Added `Button as div` example for #939

image

### CODE REVIEW

File: packages/ariakit/src/button/__examples__/button-as-div/style.css [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+    bg-primary-2
+    text-primary-2
+    hover:bg-primary-2-hover
+    dark:bg-primary-2-dark
+    dark:text-primary-2-dark
+    dark:hover:bg-primary-2-dark-hover
+    ariakit-focus-visible:ariakit-outline
+    aria-disabled:opacity-50
+  }
+
```
Comment (diegohaz) [inline packages/ariakit/src/button/__examples__/button-as-div/style.css]: I think you can just import the default styles in this case.
```suggestion
@import url("../button/style.css");

```

Comment (iamsriranga) [reply packages/ariakit/src/button/__examples__/button-as-div/style.css]: Oh, yeah, this is good 👍

Comment (iamsriranga) [reply packages/ariakit/src/button/__examples__/button-as-div/style.css]: Hey @diegohaz , take a look at this, made the change

**Review by diegohaz — APPROVED** [review]
This looks really good! Thank you @saideepesh000!

**Comment by diegohaz** [comment]
@saideepesh000 How the development experience has been so far? I know that you've had problems with the `npm run dev` command. Is it fixed? Do you think something could be different/better?

**Comment by saideepesh000** [comment]
yeah, @diegohaz , it is fixed now, I'm running the command on windows PowerShell as an admin, but we need to figure out a soln to run on normal command prompt, yesterday for some time I was using gitpod.io

**Comment by saideepesh000** [comment]
@diegohaz  and also we need to configure husky and pre-commit hooks to run prettier/eslint before pushing the code

**Comment by diegohaz** [comment]
> @diegohaz and also we need to configure husky and pre-commit hooks to run prettier/eslint before pushing the code

Oh yeah! We need that too. And maybe stylelint as well.

**Pull request merged**